lib/omniauth-wsfed/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module WSFed
-    VERSION = '0.2.3'
```

Comment:
Do we need to change the patch version here @mvsBR ?

Or we can release a new tag for the gem `0.2.3.2`.

cc: @ad-ys

Reply:
yes lets bump this to `0.2.3.2` and tag it accordingly

also i think we are actually using this branch https://github.com/yardstick/omniauth-wsfed/commits/yardstick-ruby-2.5/

Reply:
Updated the version number and created a new PR taking base of `yardstick-ruby-2.5`.

Closing this one.